test/System/Element/setElementsPinStatus.dyn

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
not opened the graph, could you attach an image of the workflow to the PR ? would take me much longer to clone it down & fire it up

Reply:
![image](https://user-images.githubusercontent.com/13732445/68853281-81924600-06d1-11ea-8ee3-a7ec3f55c10b.png)